src/app/ad-authoring/ad-authoring.service.ts

```diff
@@ -0,0 +1,42 @@
+import {Injectable} from '@angular/core';
+import {
+  AdAuthoringWorkflowState,
+  AdAuthoringWorkflowStateContainer,
+} from './ad-authoring.state';
+import {Observable} from 'rxjs';
+import {CallToActionEnum} from './call-to-action';
+import {LandingTypeEnum} from './landing-type-values';
+
+@Injectable({
+  providedIn: 'root',
+})
+export class AdAuthoringService {
+  constructor(
+    private readonly adAuthoringState: AdAuthoringWorkflowStateContainer
+  ) {}
+
+  updateLandingUrl(landingUrl: string) {
```

Comment:
Are we going to add any validation around this to make sure the user input is a valid url?

Reply:
Added url validation in preview branch